community-contributions/telegram-faucet/bot_tg_gettoque/bot.cs

```diff
@@ -0,0 +1,338 @@
+  
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using Telegram.Bot;
+using Telegram.Bot.Args;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums; //check on msg type
+using Telegram.Bot.Types.ReplyMarkups;
+using Telegram.Bot.Types.InlineQueryResults;
+using System.Net;
+using System.Net.Http;
+using System.Data.SqlClient;
+
```

Comment:
Not all of the using directives are necessary here. It would be good to delete these. Similar case in `botsender_selenium_v1.cs` as well.